app/uploaders/image_uploader.rb

```diff
@@ -86,4 +92,9 @@ def extension_whitelist
   def size_range
     0..10.megabytes
   end
+
+  def is_comment?(file)
+    width, height = ::MiniMagick::Image.open(file.file)[:dimensions]
+    model.instance_of?(Comment) && (width > 250 || height > 250)
```

Comment:
I don't think we need the dimension check here. Not only does it ambiguate what is_comment? does (it's sort of more like is_comment_and_needs_thumb_resize), but also, if someone submitted something super small, say, an 8x8 GameBoy/Atari-esque image, I'd rather still have it be resized so that it's clickable/visible. Feel free to dispute me on this point if you disagree.